spec/features/monitoring_feature_spec.rb

```diff
@@ -18,4 +18,22 @@
       expect(page).to have_selector(".nodes-container tbody tr i.fa.fa-refresh")
     end
   end
+
+  # rubocop:disable RSpec/ExampleLength
+  # rubocop:disable RSpec/MultipleExpectations:
```

Comment:
I can avoid the many lines and many expectations if I remove the first 2 "expects". In that case the test is prone to false positives. E.g. if the flash message is already there before I create the minion.